src/main/kotlin/org/pecasonline/features/items/ItemsMetricsService.kt

```
@@ -0,0 +1,27 @@
+class ItemsMetricsService(
+    private val meterRegistry: MeterRegistry
+): IItemsMetricsService {
+    override fun incrementRequests(endpoint: String, method: String, extraLabels: Map<String, String>) {
+        val labels = mutableListOf("endpoint", endpoint, "method", method)
+
+        extraLabels.forEach { (k, v) ->
+            labels.add(k)
+            labels.add(v)
+        }
+
+        meterRegistry.counter("custom.items.requests", *labels.toTypedArray()).increment()
```

detekt (1.23.8)

[warning] In most cases using a spread operator causes a full copy of the array to be created before calling a method. This may result in a performance penalty.

(detekt.performance.SpreadOperator)